{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--a6b82c4b-3411-4696-bd54-245edee9f394",
    "created": "2024-08-13T16:04:30.089031Z",
    "modified": "2024-08-13T16:04:30.089036Z",
    "name": "No Title Available",
    "description": "LibreSSL 2.5.1 to 2.5.3 lacks TLS certificate verification if SSL_get_verify_result is relied upon for a later check of a verification result, in a use case where a user-provided verification callback returns 1, as demonstrated by acceptance of invalid certificates by nginx.",
    "external_references": [
        {
            "source_name": "cve",
            "external_id": "CVE-2017-8301"
        }
    ],
    "references": [
        {
            "tags": [
                "x_refsource_MISC"
            ],
            "url": "http://seclists.org/oss-sec/2017/q2/145"
        },
        {
            "tags": [
                "x_refsource_CONFIRM"
            ],
            "url": "https://github.com/libressl-portable/portable/issues/307"
        },
        {
            "tags": [
                "x_refsource_CONFIRM"
            ],
            "url": "https://trac.nginx.org/nginx/ticket/1257"
        },
        {
            "name": "98076",
            "tags": [
                "vdb-entry",
                "x_refsource_BID"
            ],
            "url": "http://www.securityfocus.com/bid/98076"
        }
    ]
}